apis/apis.json

```diff
@@ -436,8 +436,8 @@
     "tags": [ "Spanner" ],
     "dependencies": {
       "Google.LongRunning": "1.0.0",
-      "Google.Api.Gax.Grpc": "2.2.1",
-      "Grpc.Core": "1.7.1",
+      "Google.Api.Gax.Grpc": "2.3.0",
+      "Grpc.Core": "1.10.0",
       "Google.Cloud.Iam.V1": "1.0.0"
```

Comment:
Wouldn't all these versions need to get updated too since Google.Cloud.Iam.V1 was bumped to 1.1.0-beta00? Or is this the version pulled from NuGet?

Reply:
When versions are specified explicitly, they'll be loaded from NuGet rather than the local project. If we release an IAM 1.1.0, we can update these too.

Reply:
Just curious: what criteria are used to decide whether to use a NuGet reference as opposed to a local project reference?

Reply:
If the API is GA, we need to pin all its dependencies to specific versions. In theory, everything that I've updated to a new beta00 I *could* start using the defaults/project versions again - there just didn't seem to be much benefit, as we'd probably need to pin back to the current version anyway.

We have checks that say that when we tag a release, project dependencies are only allowed if those projects are being tagged at the same time.

We've had a problem in the past where we had X depending on Y with a project dependency, Y gained a feature and X used it, then we released X without releasing Y, causing the kind of problems you'd expect...

Reply:
Interesting, makes sense. Thanks!